code/datums/quirks/neutral_quirks/transhumanist.dm

```diff
@@ -0,0 +1,147 @@
+#define MOOD_CATEGORY_TRANSHUMANIST_PEOPLE "transhumanist_people"
+#define MOOD_CATEGORY_TRANSHUMANIST_BODYPART "transhumanist_bodypart"
+// The number of silicons minus the number of organics determines the level
+#define TRANSHUMANIST_LEVEL_ECSTATIC 5
+#define TRANSHUMANIST_LEVEL_HAPPY 2
+#define TRANSHUMANIST_LEVEL_NEUTRAL 0
+#define TRANSHUMANIST_LEVEL_UNHAPPY -2
+#define TRANSHUMANIST_LEVEL_ANGRY -5
```

Comment:
I personally think the levels should be a smidge more "lenient" to the amount of organic mobs, which are always many more than the number of robotic buddies around. Just a smidge because this is a neutral quirk.